Common/XGBoost/src/main/java/org/tribuo/common/xgboost/XGBoostFeatureImportance.java

```diff
@@ -246,8 +247,6 @@ private String translateFeatureId(String xgbFeatName) {
     }
 
     public String toString() {
-        return "XGBoostFeatureImportance(" + getImportances(5).stream()
-                .map(XGBoostFeatureImportanceRecord::toString)
-                .collect(Collectors.joining(",\n\t")) + ")";
+        return String.format("XGBoostFeatureImportance(booster=%s, featureIdMap=%s)", booster.toString(), featureMap.toString());
```

Comment:
I think it might be better to return a toString that refers to the model provenance? The booster and featureIdMap toStrings aren't very helpful.